.eslintrc.js

```diff
@@ -0,0 +1,65 @@
+module.exports = {
```

Comment:
Follow gts:
https://github.com/googleapis/google-api-nodejs-client/blob/master/.eslintrc.json

Reply:
Please elaborate. `gts` is listed in the `extends`

Reply:
Oh.

Only use `gts`. The file should be:

```json
{
  "extends": "./node_modules/gts"
}
```

Any errors from this, we'll need to fix.

Reply:
Will do in due time